portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -24,9 +24,34 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  private ArrayList<String> messages = new ArrayList<>();
+  
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
+    // response.setContentType("text/html;");
```

Comment:
Please delete commented code from this commit! Note that it's possible to make a branch/tag first if it's desirable to keep a reference to this snapshot

Reply:
Got it!